packages/evm/test/validators/SignerValidator.t.sol

```diff
@@ -204,6 +207,54 @@ contract SignerValidatorTest is Test {
         validator.validate(boostId, incentiveId, claimant, claimData);
     }
 
+    function testValidate_FuzzValidInputs(
+        uint256 boostId,
+        uint256 incentiveId,
+        uint256 pk,
+        uint8 incentiveQuantity,
+        address claimant,
+        bytes memory incentiveData
+    ) public {
+        incentiveQuantity = uint8(bound(incentiveQuantity, 1, 7));
```

Comment:
This should work up to 8, right? It's the ID that needs to be under 8 necessarily